src/core/ActivityInfoTree.cpp

```diff
@@ -290,6 +290,10 @@ void ActivityInfoTree::init()
                                    "/data2/" + QString("full-%1.rcc").arg(COMPRESSED_AUDIO)))
         qDebug() << "Registered the pre-download " << QString("full-%1.rcc").arg(COMPRESSED_AUDIO);
 
+    if(QResource::registerResource(QStandardPaths::writableLocation(QStandardPaths::CacheLocation) +
+                                   "/data2/" + QString("backgroundMusic-%1.rcc").arg(COMPRESSED_AUDIO)))
+        qDebug() << "Registered the pre-download " << QString("backgroundMusic-%1.rcc").arg(COMPRESSED_AUDIO);
+
     qmlRegisterSingletonType<QObject>("GCompris", 1, 0, "ActivityInfoTree", menuTreeProvider);
```

Comment:
This does not belong here. Background-music is not pre-downloaded, it is downloaded dynamically as is wordset. Drop it here and start download in checkWordset() in main.qml via DownloadManager